pkg/operator/ceph/object/spec.go

```diff
@@ -154,11 +154,24 @@ func (c *clusterConfig) makeRGWPodSpec(rgwConfig *rgwConfig) (v1.PodTemplateSpec
 		ServiceAccountName: serviceAccountName,
 	}
 
+	rgwOpsLogEnabled := false
+	if opsLogSidecar := c.store.Spec.Gateway.OpsLogSidecar; opsLogSidecar != nil {
+		rgwOpsLogEnabled = true
+		// Generate ops log file name based on rgw daemon ID
+		opsLogFile := fmt.Sprintf("ops-log-%s", getDaemonName(rgwConfig))
+
+		// Add the side-car container named ops-log
+		shareProcessNamespace := true
+		podSpec.ShareProcessNamespace = &shareProcessNamespace
```

Comment:
Why is shared PID namespacing required? If it's no longer required, please remove. If it is, please add a comment to explain why it's needed.

From looking at k8s documentation, I don't believe this is necessary. [This example](https://kubernetes.io/docs/concepts/cluster-administration/logging/#streaming-sidecar-container) does not include the config.